pkgs/by-name/qq/qqsp/package.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  adwaita-qt,
+  libadwaita,
+  adw-gtk3,
+  gst_all_1,
+  gnome-themes-extra,
+  librsvg,
+  fetchgit,
+  qt5,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  name = "qqsp";
+  version = "1.9";
+  src = fetchFromGitHub {
+    owner = "Sonnix1";
+    repo = "Qqsp";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-eDgoa+/dcJ8Ti+YLHgKUKus0+zRrFEuJ19wUpbFpcBU=";
+  };
+  buildInputs = [
+    adwaita-qt
+    libadwaita
+    adw-gtk3
+    gnome-themes-extra
+    gst_all_1.gst-plugins-good
+    gst_all_1.gst-plugins-ugly
+    qt5.qtmultimedia
+    qt5.qtwebengine
```

Comment:
Just curious: no qt5.qtbase?

Reply:
The program built and ran fine without it, however I do see it listed as a dep so I've added it just in case